files/Aboot/boot0.j2

```diff
@@ -122,10 +125,16 @@ platform_specific() {
         aboot_machine=arista_7050_qx32s
         echo "modprobe.blacklist=radeon" >>/tmp/append
     fi
-    if [ "$sid" = "Upperlake" ]; then
+    if [ "$sid" = "Upperlake" ] || [ "$sid" = "UpperlakeES" ]; then
```

Comment:
This is just a variant of the Upperlake so it will still be 4G.
On the other hand the Gardena one will have 30G of flash. I will therefore increase the varlog size for this one.